src/tests/ftest/util/command_utils.py

```diff
@@ -124,21 +124,31 @@ def with_exports(self):
         """
         return " ".join([self.env.to_export_str(), str(self)]).strip()
 
-    def run(self):
+    def run(self, raise_exception=None):
```

Comment:
This created a bunch of pylint errors for classes that override this method due to different parameter names and whilst pylint does pick it up we didn't gate the PR on it as we only fail PRs for failures in the modified code, not failures that are created elsewhere.

Reply:
Running pylint locally on the commit preceding this one, and again on this commit seems to show the following pylint issue was added by the commit of this PR
src/tests/ftest/util/fio_utils.py:172:12: E1121: Too many positional arguments for method call (too-many-function-args)